src/ima.js

```diff
@@ -447,6 +490,9 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
         this._contentSrc = selectedSource[0].url;
       }
     });
+    this.eventManager.listenOnce(this.player, this.player.Event.DURATION_CHANGE, () => {
+      this._contentDuration = this.player.duration;
```

Comment:
init it with a default value and reset on `RESET`

Reply:
I added